inspirehep/modules/workflows/tasks/upload.py

```diff
@@ -50,7 +51,9 @@ def store_record(obj, *args, **kwargs):
     record = InspireRecord.create(obj.data, id_=None)
 
     # Create persistent identifier.
-    inspire_recid_minter(str(record.id), record)
+    obj_uuid = str(record.id)
+    inspire_recid_minter(obj_uuid, record)
+    inspire_texkey_minter(obj_uuid, record)
```

Comment:
This is what came out from my search:
``` bash

inspirehep/modules/migrator/tasks/records.py
296:            record.commit()

inspirehep/modules/disambiguation/records.py
140:    record.commit()

inspirehep/modules/disambiguation/tasks.py
175:            record.commit()

inspirehep/modules/workflows/tasks/upload.py
73:    record.commit()

inspirehep/modules/records/tasks.py
60:            record.commit()
149:            record.commit()

tests/integration/test_records_receivers.py
55:    record.commit()
```
I am pretty sure that we have to handle the texkey in the following places:
- inspirehep/modules/workflows/tasks/upload.py
- inspirehep/modules/migrator/tasks/records.py

Do you think we have to handle texkeys also in the other places?